p2p/src/main/kotlin/maru/p2p/MaruPeer.kt

```diff
@@ -38,16 +38,14 @@ import tech.pegasys.teku.networking.p2p.rpc.RpcStreamController
 interface MaruPeer : Peer {
   fun getStatus(): Status?
 
-  fun sendStatus(): SafeFuture<Unit>
```

Comment:
Why did you remove this from the public interface? Is this because updateStatus could be used instead?

Reply:
Yes, it sendStatus() is only needed internally now.